bin/deploy

```diff
@@ -4,11 +4,27 @@
 #
 #   APP defaults to carnival-staging
 #
+#   Set BUILD=1 to perform a Halcyon build
+#
 ###
 set -e
 
 case "${1:-carnival-staging}" in
-  carnival-staging) git push staging master ;;
+  carnival-staging)
+    git push --force staging master
+
+    if [ "$BUILD" = 1 ]; then
```

Comment:
Any reason to look for `1` and not just a non-empty value? I prefer to use `BUILD=OHYEAH`.